tests/drivers/adc/adc_error_cases/boards/nrf54h20dk_nrf54h20_cpuppr.overlay

```diff
@@ -0,0 +1,11 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * Copyright (c) 2025 Nordic Semiconductor ASA
+ */
+
+#include "nrf54h20dk_nrf54h20_common.dtsi"
+
+&adc {
+	status = "okay";
```

Comment:
Don't you need to also reserve it in the vpr_launcher overlay, as in other tests?